go/toolchains.rst

```diff
@@ -601,7 +601,7 @@ Fields
 | The suffix to use for shared libraries in this build mode. Mostly used when                      |
 | generating output filenames of binary rules.                                                     |
 +--------------------------------+-----------------------------------------------------------------+
-| :param:`crosstool`             | :type:`list of File`                                            |
+| :param:`cc_toolchain_files`    | :type:`list of File`                                            |
```

Comment:
As much as I dislike this fact, this is public API. It's probably still fine to make this change, but we should remember to mention it in the release notes.

Reply:
Yeah. I am highly skeptical that anyone is using this API.

I briefly considered marking it as deprecated, and adding a new attribute called `cc_toolchain_files`. But I think that would still result in the same "potential" breakages.

I think it's better just to rip off the bandage here so that the rules are cleaner and easier to maintain. If there are downstream complains about this, we could take note for the next time we do these cleanups.